apps/launchpad/app/routes/_app+/discover.tsx

```diff
@@ -146,14 +146,14 @@ export default function Discover() {
       onAction: () => window.open('https://www.hellospoc.com/', '_blank'),
       imageUrl: '/images/discover/spoc.png',
     },
-    {
-      id: '8',
-      title: 'ValueSys',
-      description: 'Community values platform by ConsenSys',
-      buttonText: 'Learn More',
-      onAction: () => window.open('#', '_blank'),
-      imageUrl: '/images/discover/valuesys.png',
-    },
+    // {
```

Comment:
I wonder if we want to drop this entirely from even being commented out until next week